.github/workflows/build.yml

```diff
@@ -3,26 +3,26 @@ on: [pull_request, workflow_dispatch]
 jobs:
   cocoapods:
     name: CocoaPods
-    runs-on: macOS-latest
+    runs-on: macOS-11.0
```

Comment:
We could probably keep this as `macOS-latest` since once Big Sur is available on GH Actions, it'll be the latest version.

Reply:
I was trying `macOS-11` since this [notice](https://github.com/actions/virtual-environments):
> The macOS 11.0 virtual environment is currently provided as a private preview only.
The "macos-latest" YAML workflow label still uses the macOS 10.15 virtual environment.

But apparently we can't access macOS-11. I _could_ see them making macOS-11 public before they make it the default version. But we can see!